main.tex

```diff
@@ -1484,29 +1487,21 @@
 
 \begin{enumerate}[label=(\alph*)]
 
-\item  obtain the Owners Corporation prior written consent before an Occupier installs any equipment on the Owner’s Lot;
-
-\item  obtain the Owners Corporation prior written consent before any heavy equipment is brought on to the Owner’s Lot;
-
-\item  ensure all equipment installed on the Owner’s Lot must be reasonably necessary for carrying on the Occupier’s business in the Lot;
+\item  obtain the Owners Corporation prior written consent before any heavy machinery is brought on to the Owner’s Lot;
 
 \item  not have any equipment that may cause any structural or other damage to the floors or other parts of the Owner’s Lot or the Common Property or nuisance (including noise or vibration) to any other Lot Owner or Occupier;
 
-\item  give the Owners Corporation at least 5 Business Days’ prior written notice before the Lot Owner brings equipment onto the Lot or the Common Property;
-
-\item  obey the Owners Corporation’s reasonable requirements in relation to the routing, installation and location of the equipment.
-
 \end{enumerate}
 
 \item  An Owner or Occupier of a Lot must:
 
 \begin{enumerate}[label=(\alph*)]
 
-\item  comply with the Owners Corporation requirements concerning the mechanical services in the Building and must not do anything which might interfere with their operation;
+\item  comply with the Owners Corporation requirements concerning the hydrological, electrical and mechanical services in the Building and must not do anything which might interfere with their operation;
 
 \item  must promptly notify the Owners Corporation if it is suspected that the service in question is not functioning satisfactorily; and
 
-\item  not use the water, lighting, heating and fire sprinkler systems in the Lot for any purpose other than those for which they were constructed.
```

Comment:
If someone uses a light in a way differently to "the purpose for which it was constructed", what could that even be? And how is it any of the Owners Corporation's business?